pkg/apis/core/validation/validation.go

```diff
@@ -1533,6 +1533,23 @@ func validateStorageOSPersistentVolumeSource(storageos *core.StorageOSPersistent
 	return allErrs
 }
 
+// ValidatePVSecretReference check whether provided SecretReference object is valid in terms of secret name and namespace.
+
+func ValidatePVSecretReference(secretRef *core.SecretReference, fldPath *field.Path) field.ErrorList {
```

Comment:
start with this unexported to avoid picking up additional callers outside the package

Reply:
@liggitt sure, unexported this func as a start..thanks ..ptal.